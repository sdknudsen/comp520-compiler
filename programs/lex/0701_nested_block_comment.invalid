/* This is a nested /* block comment */. It should fails */

