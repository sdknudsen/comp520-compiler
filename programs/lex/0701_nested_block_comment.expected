DOT
ID(It)
ID(should)
ID(fails)
TIMES
DIV
EOF
